README.md

```diff
@@ -121,6 +121,11 @@ To try this build. In the settings of this repo in you drone ui. Set the path fo
 
 For more advanced information on services go here `https://docs.drone.io/pipeline/kubernetes/syntax/services/`
 
+### Cards
+
+Here is an example of creating cards using Drone-Docker, Snyk & Git-Leaks plugin.
```

Comment:
```
Here is an example of creating cards using Drone-Docker, Snyk & Git-Leaks plugin.
To try this build. In the settings of this repo in you drone ui. Set the path for the drone file to `./cards/card.yml`
```
Here is an example of a build that displays cards. It utilizes the Drone-Docker, Snyk & Git-Leaks plugins.

sounds better.

You should also note that the user has to setup passwords / users.